packages/components/src/internal/util/helpLinks.tsx

```diff
@@ -41,15 +41,33 @@ export const SAMPLE_ALIQUOT_TOPIC = 'aliquot';
 
 export const UNIQUE_IDS_TOPIC = 'uniqueStorageIds';
 
-export function getHelpLink(topic: string): string {
-    return getServerContext().helpLinkPrefix + topic;
+export function getHelpLink(topic: string, referrer = 'inPage'): string {
+    return getServerContext().helpLinkPrefix + topic + '&referrer=' + referrer;
 }
 
-// TODO: This should be converted into a React.FC with arguments switched to props
+interface HelpLinkProps {
+    className?: string;
```

Comment:
I would recommend `className?: React.CSSProperties;` as a more explicit type.

Reply:
If I do that I get typescript errors for the `className` attribute of the `<a>` tag:
```
ERROR in /Users/corynathe/LabKey/labkey-ui-components/packages/components/src/internal/util/helpLinks.tsx(58,64)
      TS2322: Type 'CSSProperties' is not assignable to type 'string'.
```